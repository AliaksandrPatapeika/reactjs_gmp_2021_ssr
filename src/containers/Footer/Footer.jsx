import './Footer.less';

import React from 'react';

import Logo from '../../components/Logo';

const Footer = () => (
  <footer className="footerContainer">
    <Logo />
  </footer>
);

export default Footer;
