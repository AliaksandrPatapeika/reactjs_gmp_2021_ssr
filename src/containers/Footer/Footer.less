@import "../../less/variables";

.footerContainer {
	display: flex;
	justify-content: center;
	padding: 26px 0;
	background-color: @dark-gray;
}