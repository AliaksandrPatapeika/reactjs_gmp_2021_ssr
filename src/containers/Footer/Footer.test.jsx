import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './Footer';

describe('Footer', () => {
  test('renders correctly', () => {
    const footer = renderer
      .create(<Footer />)
      .toJSON();

    expect(footer).toMatchSnapshot();
  });
});
