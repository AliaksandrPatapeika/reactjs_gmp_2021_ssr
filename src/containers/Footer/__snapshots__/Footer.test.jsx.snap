// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="footerContainer"
>
  <div
    className="logoContainer"
  >
    <span
      className="netflix"
    >
      netflix
    </span>
    <span
      className="roulette"
    >
      roulette
    </span>
  </div>
</footer>
`;
